source/Classroom/Troubleshooting/Authentication/list_upload_troubleshooting.md

```diff
@@ -1,53 +1,100 @@
 ---
 seo:
   title: List Upload Troubleshooting
-  description: List Upload Troubleshooting
-  keywords: format, list, header, upload, error, UTF-8
+  description: Having trouble uploading a list in Marketing Campaigns? Learn common solutions here...
+  keywords: marketing, campaigns, csv, upload, contacts, format, list, header, error, UTF-8
 title: List Upload Troubleshooting
 weight: 0
 layout: page
-zendesk_id: 206317698
 navigation:
   show: true
 ---
 
- **My upload file is not accepted**
+{% info %}
+In the case of an error, we will often send an email to you in the case of a list upload failure. Please check the email from SendGrid that outlines the results of your CSV upload, including a downloadable error report.
```

Comment:
In the case of a list upload failure, you may receive an email from SendGrid. Please check the email from SendGrid containing the results of your CSV upload, including a downloadable error report.